api/handlers/exporter.go

```diff
@@ -54,22 +81,32 @@ func (e *Exporter) Decideds(w http.ResponseWriter, r *http.Request) error {
 	from := phase0.Slot(request.From)
 	to := phase0.Slot(request.To)
 
+	var pubkeys []spectypes.ValidatorPK
+	for _, req := range request.PubKeys {
+		var pubkey spectypes.ValidatorPK
+		if len(req) != len(pubkey) {
+			return api.BadRequestError(errors.New("invalid pubkey length"))
```

Comment:
I think it makes sense to return the problematic pubkey or at least an index, to help someone that created a big request find the issue.

```suggestion
	for i, req := range request.PubKeys {
		var pubkey spectypes.ValidatorPK
		if len(req) != len(pubkey) {
			return api.BadRequestError(fmt.Errorf("invalid pubkey length at index %d", i))
```